src/main/java/nl/tudelft/ti2206/group9/util/GameObserver.java

```diff
@@ -63,8 +63,12 @@
         START,
         /** Settings button has been pressed. */
         SETTINGS,
-        /** Setting "sound" has been toggled. optionalArg: isSoundEnabled. */
-        SETTING_SOUND,
+        /** Setting "soundtrack" has been toggled.
+         * optionalArg: isSoundtrackEnabled. */
+        SETTING_SOUNDTRACK,
+        /** Setting "sound effects" has been toggled.
+         * optionalArg: isSoundEffectsEnabled. */
+        SETTING_SOUNDEFFECTS,
```

Comment:
I would prefer the optionalArg to be defined as `whether sound is enabled ("ON"/"OFF")`. Now, it looks like a boolean can be passed, which is not the case.

<review, documentation